osu.Game.Rulesets.Mania/ManiaSettingsSubsection.cs

```diff
@@ -44,7 +44,12 @@ private void load()
                     Keywords = new[] { "color" },
                     LabelText = RulesetSettingsStrings.TimingBasedColouring,
                     Current = config.GetBindable<bool>(ManiaRulesetSetting.TimingBasedNoteColouring),
-                }
+                },
+                new SettingsEnumDropdown<ManiaMobilePlayStyle>
```

Comment:
Probably should be hidden on desktop.

Reply:
I've kept it visible for laptops with touchscreen support but maybe better hiding it altogether?

Reply:
I've went ahead and guarded it against mobile check for simplicity purposes.

Reply:
https://github.com/ppy/osu/pull/32314/commits/b64e69d581a08d0be9d5e705f9bb1078faf3050f#diff-32bd19fe4c9e65398af40ae184893910eb856e4bdf5ed89649cb799c931f49adR51